moped-editor/src/utils/activityLogFormatters/mopedFundingActivity.js

```diff
@@ -11,33 +24,25 @@ export const formatFundingActivity = (
 
   const changeIcon = <MonetizationOnOutlinedIcon />;
 
+  const newRecord = change.record_data.event.data.new;
+  const oldRecord = change.record_data.event.data.old;
+
+  const fundingSourceText = getFundingSourceIdText(
+    newRecord,
+    fundingSources,
+    fundingPrograms
+  );
+
   // add a new funding source
   if (change.description.length === 0) {
     // if the added record has a funding source, use that as the change value
-    if (change.record_data.event.data.new.funding_source_id) {
+    if (fundingSourceText) {
       return {
         changeIcon,
         changeText: [
           { text: "Added ", style: null },
           {
-            text: fundingSources[
-              change.record_data.event.data.new.funding_source_id
-            ],
-            style: "boldText",
-          },
-          { text: " as a new funding source.", style: null },
-        ],
-      };
-      // if not, then check if theres a funding program
-    } else if (change.record_data.event.data.new.funding_program_id) {
-      return {
-        changeIcon,
-        changeText: [
-          { text: "Added ", style: null },
-          {
-            text: fundingPrograms[
-              change.record_data.event.data.new.funding_program_id
-            ],
+            text: fundingSourceText,
             style: "boldText",
           },
           { text: " as a new funding source.", style: null },
```

Comment:
Should the period at the end of the text stick around? Just noticed that other changes don't have one. I could have missed others that do, though.

Reply:
this is the only one I can see with a period at the end. good catch.